nx-X11/lib/X11/XlibInt.c

```diff
@@ -431,28 +420,57 @@ _XWaitForWritable(
 #endif
 
 #ifndef NX_TRANS_SOCKET
-#  ifdef USE_POLL
+#ifdef USE_POLL
 	    filedes.events = POLLIN;
 	filedes.events |= POLLOUT;
-#  else
+#else
 	FD_SET(dpy->fd, &r_mask);
         FD_SET(dpy->fd, &w_mask);
-#  endif
+#endif
 #endif /* #ifndef NX_TRANS_SOCKET */
 
 	do {
+#ifdef NX_TRANS_SOCKET
```

Comment:
Why does this commit touch XlibInt.c at all? Why does NX'ish code get re-added in this commit. As I understand it, you basically have to commits for the 1.3.4 version bump.

One commit brings all 1.3.4 updates into files that had been modified for NX back then. And in the second commit (this one), you copy in all files from 1.3.4 that never had to be adapted for NX. So, this second commit should not touch the XlibInt.c file (all 1.3.4 backports should be in the first commit, referenced above).

Right?